django/test/selenium.py

```diff
@@ -118,6 +161,51 @@ def setUpClass(cls):
         super().setUpClass()
         cls.addClassCleanup(cls._quit_selenium)
 
+    @contextmanager
+    def desktop(self):
+        with ChangeWindowSize(1280, 720, self.selenium):
+            yield
+
+    @contextmanager
+    def small_screen(self):
+        with ChangeWindowSize(1024, 768, self.selenium):
+            yield
+
+    @contextmanager
+    def mobile(self):
+        with ChangeWindowSize(360, 800, self.selenium):
+            yield
+
+    @contextmanager
+    def rtl(self):
+        get_language_bidi_mock = mock.patch(
+            "django.utils.translation.get_language_bidi", return_value=True
+        )
+        get_language_bidi_mock.start()
+        with self.desktop() as desktop:
+            try:
+                yield [desktop]
+            finally:
+                get_language_bidi_mock.stop()
+
+    @contextmanager
+    def dark(self):
+        # Need to navigate to a page before executing a script.
+        self.selenium.get(self.live_server_url)
+        self.selenium.execute_script("localStorage.setItem('theme', 'dark');")
```

Comment:
I forget with selenium... But do we need to wait for anything to load before executing the script here? Or does it work fine?

Reply:
It appears to be working but maybe this is something I can do instead of navigating to a url 🤔